.github/workflows/release.yml

```diff
@@ -40,7 +40,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN || github.token }}
         uses: changesets/action@v1
         with:
-          version: node scripts/version-alpha.mjs
```

Comment:
We now use the default changeset behaviour

Reply:
We should be able to delete the `scripts/version-alpha.mjs` file now

Reply:
Agreed. I intend to do a general cleanup next week.